NewForce-Cohort-5/tabloidfullstack-the-pollening
# Mel tag 13

# Description
Ticket #13 View all Tags

Ticket #13, View all tags - For this feature I created a Tag repository, ITag repository, Tag controller, Tag.cs model in Visual studio 2020. Also in visual studio code I created a tag component with Tag.js, TagList.js, along with a Tag Provider to get all the tags with a get method. A button for tag Management has has been created too in the Nav Bar. 

Fixes # (issue)

## Type of change




# Testing Instructions

This test requires you to have data in your Tabloid db. If you do not already, please run the following queries so you have SQL seed data (these can be found in the /SQL directory):

[SQL/01_Tabloid_Create_DB.sql](https://github.com/NewForce-Cohort-5/tabloidfullstack-the-pollening/blob/main/SQL/01_Tabloid_Create_DB.sql)

[SQL/02_Tabloid_Seed_Data.sql](https://github.com/NewForce-Cohort-5/tabloidfullstack-the-pollening/blob/main/SQL/02_Tabloid_Seed_Data.sql)

After creating your seed data, use any of the emails currently in the UserProfile table to test. You must first login with this email, 
then go Go to Tag Management on the NavBar, then you should see all the available tags. 




# Checklist:
